### ISSUE TITLE
Adding custom modded support through json creates fails

### ISSUE THREAD

**Issue by WenXin20** [Aang23/UndergroundBiomesConstructs#165]
Minecraft v1.14.4
Forge v28.1.1
UBC v1.0-47

After creating json files and overlays for druidcraft, I realized that UBC doesn't recognize the ore and will instead create an ore for minecraft air and a block called `'ubcstone_name' null` is created. Did I format something wrong or is there an error within the mod?

The wiki is also outdated and doesn't mention how the format works for 1.14.4
[druidcraft.txt](https://github.com/Aang23/UndergroundBiomesConstructs/files/3612151/druidcraft.txt)

Log Excert:
```
[18:15:39] [Server-Worker-6/WARN]: Exception loading blockstate definition: undergroundbiomes:blockstates/metamorphic_ore_minecraft_air_greenschist.json: java.io.FileNotFoundException: undergroundbiomes:blockstates/metamorphic_ore_minecraft_air_greenschist.json
[18:15:39] [Server-Worker-6/WARN]: Exception loading blockstate definition: 'undergroundbiomes:blockstates/metamorphic_ore_minecraft_air_greenschist.json' missing model for variant: 'undergroundbiomes:metamorphic_ore_minecraft_air_greenschist#'
```

**Comment by Mahtaran** [comment]
Well, apart from the fact that the file you shared ends with the `.txt` extension (which I hope/suspect is not the case with the file you use), it does seem to follow the correct format (default format to exemplify)
```json
{
    "minecraft:iron_ore": {
        "overlay": "undergroundbiomes:block/overlays/minecraft/iron",
        "variant": "stone",
        "name": "Iron Ore"
    }
}
```
Where is your file located? And do you have the correct overlay textures?

**Comment by WenXin20** [comment]
Yes, the file is a `.json` file, but since GitHub doesn't accept json files, I had to change it to a `.txt` file.
The file is located at `config/undergroundbiomes/ores`
And the overlay textures are at
`config/undergroundbiomes/overlays`

The overlay textures are working, but only the Amber Ore texture was used since the game could only create one 'air ore'.

**Comment by Mahtaran** [comment]
That is really weird. @Aang23 any clues?

**Comment by Aang23** [comment]
That's in fact pretty weird. @WenXin20 Could you pack & upload your whole config folder into a .zip or something? Also I may need your logs. *Gonna try myself*
Though I see a typo here : `config/undergroundbiomes/overlays/driudcraft`

**Comment by WenXin20** [comment]
The typo here was from typing it out by hand on my phone.
Here's the [config.zip](https://github.com/Aang23/UndergroundBiomesConstructs/files/3613879/config.zip) and the [ubpack.zip](https://github.com/Aang23/UndergroundBiomesConstructs/files/3613897/ubpack.zip) that was automatically created. I'll have the full log soon once my game loads back up.

**Comment by Mahtaran** [comment]
Do the other configurations for other mods work?

**Comment by WenXin20** [comment]
No, none of the modded ones work.

**Comment by Aang23** [comment]
That's weird. I had made such it worked a while back and never had any issues.
Though according to your ubpack.zip, my pack generator didn't actually generate anything for any ores but vanilla.
Could I get your logs?

**Comment by WenXin20** [comment]
Had trouble uploading it to gist.github.com, so here is a zip file of it instead: [latest.zip](https://github.com/Aang23/UndergroundBiomesConstructs/files/3614022/latest.zip)

**Comment by Aang23** [comment]
Well your archive is empty...
Try https://paste.altillimity.com?
Or hatebin.com?

**Comment by WenXin20** [comment]
Whoops, this time it should contain it: [latest.zip](https://github.com/Aang23/UndergroundBiomesConstructs/files/3614029/latest.zip)

**Comment by Aang23** [comment]
Well that's.. Even weirder it didn't even load anything.
This is definitely a bug.

**Comment by WenXin20** [comment]
That's really weird then...

**Comment by WenXin20** [comment]
I've also noticed a very minor bug. For Black and Red Granite ores, the word "granite" is lowercase for the "g" in the tooltip, so it looks a bit odd.
Example: Black granite Gold Ore
